recipes-fsl/packagegroups/packagegroup-fsl-isp.bb

```diff
@@ -1,5 +1,7 @@
 DESCRIPTION = "Add packages for ISP build"
 
+# basler-camera* gets dynamically renamed
+PACKAGE_ARCH = "${TUNE_PKGARCH}"
```

Comment:
Shouldn't it to be `MACHINE_SOCARCH`?

Reply:
I just took what what Richard proposed in his commit without thinking it through.
The packagegroup today is only used for iMX8MP, but I would think that the packagroup, i.e. the RRECOMMENDS and RDEPENDS in it would be applicable for any MACHINE_SOCARCH, even if the packages installed would have a different content depending on SOC.

Reply:
On the hand, building it per soc arch will not eat much time and if the lib* packages get a different name per SOCARCH it cannot hurt. Will change to MACHINE_SOCARCH?